gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+// Reveal problems with box-sizing set to border-box, see https://issues.sonatype.org/browse/RSC-84
+
+.nx-card__content--2-column-example {
+  align-items: center;
+  display: flex;
+
+  >:first-child {
+    margin-right: 16px;
+  }
+}
+
+.nx-card--align-left {
+  text-align: left;
```

Comment:
This one feels weird to me.  Like left aligning the title and left aligning the content text are two separate, and rather custom, concerns.  I'm not sure it would generally make sense to do them both with a single class and I'm not sure it isn't too arbitrary for us to be demonstrating.

Reply:
> This one feels weird to me. Like left aligning the title and left aligning the content text are two separate, and rather custom, concerns. I'm not sure it would generally make sense to do them both with a single class and I'm not sure it isn't too arbitrary for us to be demonstrating.

First pass I had it as separate values for each area but it looked so ugly having one centred and another left justified that I figured that no designer would ever do that. In the unlikely event that a designer does request that the developer would be welcome to create their own modifier.

It may be too arbitrary, I added it in because there was a mockup that appeared with left aligned content.